kmath-trajectory/src/commonTest/kotlin/space/kscience/kmath/trajectory/Math.kt

```diff
@@ -0,0 +1,27 @@
+package space.kscience.kmath.trajectory
+
+import space.kscience.kmath.geometry.Line2D
+import space.kscience.kmath.geometry.Vector2D
+import space.kscience.kmath.trajectory.segments.components.Pose2D
+import space.kscience.kmath.trajectory.segments.theta
+import kotlin.math.PI
+import kotlin.math.abs
+import kotlin.math.sin
+
+const val maxFloatDelta = 0.000001
+
+fun Double.radiansToDegrees() = this * 180 / PI
+
+fun Double.equalFloat(other: Double) = abs(this - other) < maxFloatDelta
+fun Pose2D.equalsFloat(other: Pose2D) = x.equalFloat(other.x) && y.equalFloat(other.y) && theta.equalFloat(other.theta)
+
+fun Line2D.inverse() = Line2D(direction, base)
```

Comment:
I think there is some misunderstanding here. The documentation in `geometry` is missing, but `direction` is vector that added to `base`, not a "second point". So line equation is `base +x*direction` Probably better to rectify it everywhere.

Reply:
Introduced the Straight segment to replace the incorrect Line2D usage